oarepo_vocabularies/ui/templates/semantic-ui/oarepo_vocabularies_ui/VocabulariesSidebar.jinja

```diff
@@ -1,12 +1,11 @@
 {#def record, extra_context #}
 <div class="rel-mt-2">
-{%- if extra_context.permissions.can_edit %}
-  <SidebarLink href={record.links.ui_links["edit"]} icon="edit">{{ _('Edit')}}</SidebarLink>
+{%- if extra_context.permissions.can_update %}
+  <SidebarLink href={{record.links["edit_html"]}} icon="edit">{{ _('Edit')}}</SidebarLink>
   {%if extra_context.vocabularyProps["hierarchical"] == True%}
-    <SidebarLink icon="plus" href={ record.links.ui_links["create"] ~ '?h-parent=' ~ record['id'] }>
+    <SidebarLink icon="plus" href={{ record.links["create"] ~ '?h-parent=' ~ record['id'] }}>
```

Comment:
Tip: we should serialize this link to parent with UI record serializer - if possible on BE. Here we should do just `record.links.parent` (similar to what RDM record has)